test/TEST-35-NUMAPOLICY/testsuite.sh

```diff
@@ -0,0 +1,289 @@
+#!/bin/bash
+set -ex
+set -o pipefail
+
+at_exit() {
+    if [ $? -ne 0 ]; then
+        # We're exiting with a non-zero EC, let's dump test artifacts
+        # for easier debugging
+        [ -f "$straceLog" ] && cat "$straceLog"
+        [ -f "$journalLog" ] && cat "$journalLog"
+    fi
+}
+
+trap at_exit EXIT
+
+systemd-analyze log-level debug
+systemd-analyze log-target journal
```

Comment:
Okay, regarding the @keszybz's comment, it would be wise to somehow modify the rest of the test so we can switch this to `console` to make the debugging much easier, as the Ubuntu CI doesn't provide the collected journal files in case of a test fail. After a quick glance at the man page, there doesn't seem to be a way to set the log target to both journal and console - or is there a way to do that?

Reply:
Ah, nevermind, I've already added an `EXIT` hook which should dump some useful logs :-)